waspc/data/Generator/templates/server/src/routes/index.js

```diff
@@ -17,7 +17,7 @@ const router = express.Router()
 const middleware = globalMiddlewareConfigForExpress()
 
 router.get('/', middleware, function (_req, res, _next) {
```

Comment:
Since `_next` is positional, you can drop it.

Old code I know, but improvements are always welcome :)